pkg/api/authz.go

```diff
@@ -165,7 +165,7 @@ func matchesRepo(globPatterns map[string]bool, repository string) bool {
 	return allowed
 }
 
-func AuthzHandler(ctlr *Controller) mux.MiddlewareFunc {
+func AuthzHandler(ctlr *Controller, defaultAuthz bool) mux.MiddlewareFunc {
```

Comment:
Add a comment for what "defaultAuthz" is for?
There is a authZ policy but no authN config.

Reply:
Would add a comment?